app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="heading">
+  <%= link_to(image_tag("left-arrow.svg", alt: "Back Arrow"), "javascript:history.go(-1)", class: 'back-icon') %>
+    <h2 class="page-name">TRANSACTIONS</h2>
+  <div class="empty-div"></div>
+</div>
+
+
+<section class="expense-show">
+
+  <li class="expense-item">
+    <div class="expense-link">
+      <div class="icon-container">
+        <%= image_tag @expense.icon , class: 'icon'%>
+      </div>
+      <div class="expense-name">
+        <%= @expense.name %>
+      </div>
+      <div class="total-amount">
+        $ <%= @expense.total_payment_amount %>
+      </div>
+    </div>
+  </li>
+
+  <ul class="payment-list">
+    <% @expense.allocations.each do |allocation| %>
+      <li class="expense-link payment-item">
+        <div class="expense-name">
+          <%= allocation.payment.name %>
+        </div>
+        <div class="total-amount">
+          <%= allocation.payment.amount %>
+        </div>
+      </li>
+    <% end %>
```

Comment:
- [ ] Please make your design as close as possible to the `Behance` design. You did great but still, your design needs little tweaks to match exactly as it is required. Right now, from your transactions list, it is difficult to tell whether the `transaction` list does contain the `quantity` or `price`. To make it easier for someone to understand it, please add `$` before the number. Kindly take a look at the below screenshot.

![image](https://github.com/BHObserver/budget-app/assets/66476329/76ad87c7-d5bc-460b-95f2-6897f423cdf3)